library/hawkular_alerts_group_trigger.py

```diff
@@ -134,6 +147,30 @@ def delete_group_trigger(self, group_id):
         except Exception as e:
             self.module.fail_json(msg="Failed to delete group trigger. Error: {error}".format(error=e))
 
+    def conditions_update_required(self, trigger_id, conditions):
+        """ Returns True if an update is required in one or more of the group
+            trigger conditions, False otherwise
+        """
+        tgs = self.client.get_trigger_conditions(trigger_id)
+
+        # compare number of conditions
+        if len(conditions) != len(tgs):
+            return True
+
+        # sort conditions lists by the condition name
+        tgs_sorted = sorted(tgs, key=lambda k: k.context['name'])
+        conditions_sorted = sorted(conditions, key=lambda k: k['name'])
+
+        for i in range(len(conditions_sorted)):
```

Comment:
I thought about it but decided to postpone it a bit, at least till we'll decide that this (unique naming in context) is the right approach